lisa/node.py

```diff
@@ -63,14 +66,24 @@ def __init__(
         self._support_sudo: Optional[bool] = None
         self._connection_info: Optional[ConnectionInfo] = None
 
+    def __str__(self) -> str:
+        conn = self._connection_info
+        return (
+            f"{self.name if self.name else 'unnamed'}"
+            + f"[{'remote' if self.is_remote else 'local'}]: "
+            + f"{'no conn info' if not conn else conn}"
```

Comment:
What's output with this change? We're looking on info level log carefully. Print only most useful/frequent information on info level, and make it like a story.

Reply:
Well just like how it reads:
`2021-04-08 15:58:37.754 DEBUG LISA.node[0] new node unnamed[remote]: no conn info, is_default: False`.
This does not clutter it much more, and make it easy to refer to nodes